lunavl/sdk/luna_faces.py

```diff
@@ -383,24 +383,29 @@ class VLFaceDetector:
     def __init__(
         self,
         detectorType: DetectorType = DetectorType.FACE_DET_DEFAULT,
-        faceEngine: Optional[VLFaceEngine] = None,
```

Comment:
I want set up a custom face Engine for Face Detector instance

Reply:
i did it